ci/build.yml

```
@@ -153,6 +153,22 @@ core unix frozen debug build:
     untracked: true
     expire_in: 1 week
 
+core unix frozen debug build arm:
+  image: vdovhanych/nixos
```

this should work now (new arm64 images have been pushed to docker hub recently:

```suggestion
  image: nixos/nix:latest-arm64
```

---

This is the one that is broken (nix 2.5.0) that was pushed to docker hub 3 days ago. The latest image was reverted to old nix on alpine (2.3.16) which doesn't have an arm version.

---

OK